### PR TITLE
Adding README.md and images

### PR DESCRIPTION
**Pull request by LazyKitKat**
I took some time to edit the main README file. Feel free to use it if you feel like it.

### CODE REVIEW

**Comment by dimkanovikov** [comment]
Hey! Thank you for your contribution! I'll try to give you feedback asap.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,101 @@
-# STARC
+![Starc Logo](./img/starc.png)
```
Comment (dimkanovikov) [inline README.md:1]: Can we make the logo a bit smaller? :) Or may be place it as a floating image with an introduction like [here](https://gist.github.com/DavidWells/7d2e0e1bc78f4ac59a123ddf8b74932d)

File: README.md [hunk 2]
```diff
@@ -1,3 +1,101 @@
...
 
-## Story Architect is a writer's best friend
+  
+
+# Welcome to Story Architect
+
+  
+
+Hello and welcome to **Starc's** GitHub page. Story Architect is a project created by the authors of an open source screenwriting tool [**Kit Scenarist**](https://github.com/dimkanovikov/KITScenarist) and is currently under development with its current release being alpha. It's an advanced text editor for script writers working professionally in film, theatre, radio or writing for comic books.
```
Comment (dimkanovikov) [inline README.md:9]: I suggest mentioning what concrete features are not been implemented yet to not make readers think that they are already available. Like plays, mindmaps, images, etc.

File: README.md [hunk 3]
```diff
@@ -1,3 +1,101 @@
...
+
+**Starc** is available for Windows, Mac and Linux. You can download it from [**starc.app**](https://starc.app/download) website.
+
+  
+
+## Supported formats
+
+  
+
+Starc has its own *.starc* file format, but also allows you to import files using Final Draft's *.fdx* format as well as *.docx*, Fountain and importing projects from **Kit Scenarist**.
```
Comment (dimkanovikov) [inline README.md:25]: plus .odt, .trelby & .celtx files

File: README.md [hunk 4]
```diff
@@ -1,3 +1,101 @@
...
+
+**NOTE:** This feature is in development and will be added soon.
+
+## Build instructions
+
+### Linux
+
+    git clone https://github.com/story-apps/starc
+    cd starc/build
+    ./make-starc-app-image.sh
```
Comment (dimkanovikov) [inline README.md:93]: It's not a build script :) It's just a helper script which we use for build AppImage file inside the GitHub Action.

Build instruction here should be something like

```
git clone --recurse-submodules https://github.com/story-apps/starc
cd starc
qmake && make
```

or just refer to the GitHub actions [build script](https://github.com/story-apps/starc/blob/master/.github/workflows/build_starc_app.yml).

**Review by dimkanovikov — CHANGES_REQUESTED** [review]

**Comment by LazyKitKat** [comment]
Hi, I updated it as you requested. I will follow along the Wiki pages and the official website to add more info about the current state of the project as well as the implementation of new features.

**Comment by dimkanovikov** [comment]
> Hi, I updated it as you requested. I will follow along the Wiki pages and the official website to add more info about the current state of the project as well as the implementation of new features.

Thanks! Two more things I ask you to improve 

- in the localization section please mention that the app supports any language for writing and has a number of interface translations or something similar. In my experience sometimes, when people don't see their native language in the list of interface translations they think that the app does not support it at all.
- in the build instructions let's remove the Linux subtitle, these steps not depends on any platform

**Comment by dimkanovikov** [comment]
Nice! Thanks!

**Pull request merged**